packages/react-native-renderer/src/ReactNativeBridgeEventPlugin.js

```diff
@@ -78,12 +100,28 @@ export function traverseTwoPhase(inst: Object, fn: Function, arg: Function) {
   }
   for (i = 0; i < path.length; i++) {
     fn(path[i], 'bubbled', arg);
+    // It's possible this is false for custom events.
+    if (!bubbles) {
```

Comment:
Is this related to adding support for capturable-but-not-bubbling events? I had concerns with this that I described in the post about events and composition. I'd like to make sure we have addressed these concerns before adding/exposing that feature. I'm not sure if this PR is orthogonal to what @lunaleaps tried originally, or whether it includes a similar feature. If it includes that feature, then I’d like to make sure we have @sebmarkbage’s stamp on that.

Reply:
I believe this PR is mostly orthogonal.

This is for capturable-but-not-bubbling CustomEvents since those are legal under the W3C spec.

Reply:
Oh this is actually similar to what I had tried here: https://github.com/facebook/react/pull/23366/files#diff-64c0fc0cf584da861da7ea6b3b66f74bebbc8fa4857be0c4021f186f812219ddR96

Reply:
I'm submitting a new PR that just adds addEventListener/removeEventListener, so we don't need any code related to CustomEvent for now.